Formula/twilio.rb

```diff
@@ -3,9 +3,9 @@
 class Twilio < Formula
   desc "unleash the power of Twilio from your command prompt"
   homepage "https://github.com/twilio/twilio-cli"
-  url "https://github.com/twilio/homebrew-brew/releases/download/v0.0.3/twilio-v1.2.0.zip"
-  version "1.2.0"
-  sha256 "d698cf52e16084431319db5d79f9b794e1f6a3be8ce6d08c8304e54cb6c88175"
+  url "https://twilio-cli-prod.s3.amazonaws.com/twilio.tar.gz"
```

Comment:
This should be the versioned asset: `https://twilio-cli-prod.s3.amazonaws.com/twilio-v1.2.1/twilio-v1.2.1.tar.gz`